blog-posts/boosting-developer-productivity-with-github-actions.md

```diff
@@ -0,0 +1,266 @@
+
+With that set up we now define the jobs that make up this workflow (6). This workflow consists of only a single job that is called `commitlint-pr-title`, or, to make it more human-readable we can give it a "name" property, like "Ensure proper PR title" (7). This job demands to be run on a runner that is labeled `ubuntu-latest` (8). The job is made up of a list of steps (9). The first step `uses` an action called `actions/checkout` (10). This action will do a lightweight clone of the repository on the runner. Judging by the name of this action you can infer that it can be found at https://github.com/actions/checkout. All actions that start with `actions/` are part of the "official" actions and are maintained by GitHub. The next step also uses one of the official Actions called `actions/setup-node` and provides an "input" for this action using the `with` property (12) to pass the specific `node-version` that we want installed on this runner (13).
+
+While the previous two steps were using Actions, the next step runs a CLI command to install a dependency using the `run` property instead of `uses` that we've used in the steps before (14). We will need this dependency in the final step of this job. To improve the readability of the workflow output we explicitly give this step a name (15). For the steps that don't receive a `name` property, the name in the workflow output will simply be the name of the action or the raw CLI command that was run.
+
+The final step in this job is using another action, called `satellytes/commitlint-pr-title` (16).
+
+You might have noticed that whenever we use an action as a step we also specified some kind of version identifier alongside the name of the action: `actions/checkout@v2`. The name of the action is `actions/checkout` and the `@v2` in this case points to the tag `v2` in the repository of that action. These version identifiers can be any [Git Reference](https://git-scm.com/book/en/v2/Git-Internals-Git-References) that you want. It can be a branch name, a commit hash or a tag. The recommendation is to keep these as precise as possible, so ideally you would avoid having a very imprecise "version" like `@main`.
+
+This workflow will produce an output like this:
+![The output of the workflow](./images/github-actions/workflow-output.jpg)
+
+### Workflows in a GitHub repository
+All the workflows in your repository can be found in a special tab called "Actions":
+![The Actions tab](./images/github-actions/actions-tab.jpg)
+
+In this tab you can take a look at any of the past or ongoing workflow runs and their outputs and results.
+
+Workflows that are run as part of a PR will additionally appear in the "Checks" tab in a Pull Request:
+![The Checks tab in a Pull Request](./images/github-actions/checks-tab.jpg)
+
+## Actions
```

Comment:
I agree that this could very well be split up. But it feels like an artificial split and I'd prefer to keep it as a single blog post, to be honest.
If necessary, readers can take a break and continue reading after a while... besides being able to stay focused, what would you consider benefits of splitting this up?

Reply:
No, I'm good. I just wanted to know your thoughts about the split. It reads very well right now and it has the necessary cohesion 👍